assignments/src/cs/assignment01/RandomWalker.java

```diff
@@ -0,0 +1,27 @@
+package cs.assignment01;
+
+public class RandomWalker {
+    public static void main(String[] args) {
+        int n = Integer.parseInt(args[0]);
+        int t = Integer.parseInt(args[1]);
```

Comment:
Read the requirement carefully. The `RandomWalk` program takes only **one** argument.